tasks/sep/031-pectra-defense/README.md

```diff
@@ -0,0 +1,45 @@
+# ProxyAdminOwner - Set Dispute Game Implementation
+
+Status: READY TO SIGN
+
+## Objective
+
+This task updates deploys new Fault Dispute games with upgrade prestate hashes. This updated
+prestate hash corresponds with the hardfork to support the L1 Pectra upgrade.
+
+## Input data derivation:
+
+The OPPrestateUpdater contract was deployed in advance to [0xd827a8d74fef6d92b0a13bb73a32c7c3ec2626a1](https://sepolia.etherscan.io/address/0xd827a8d74fef6d92b0a13bb73a32c7c3ec2626a1).
```

Comment:
Any way I can verify this is the right OPPU version? Even if it's just a discord message or something

Reply:
I've had trouble verifying it on etherscan. Maybe checkout [this commit](https://github.com/ethereum-optimism/optimism/commit/561d9ac73d9d0883c0636c8eb73e7df80ad75d40), then run

```
forge script scripts/deploy/DeployOPPrestateUpdater.s.sol:DeployOPPrestateUpdater
```

I believe it should try to deploy to the same address.

Reply:
That gave me a deploy address of `0xcF818b7407755b2dA91B0bc9462303980C062BFC`, which matches what is in the README (changes since when I originally left this comment). We defintely should make sure to verify on sepolia and mainnet though, and can ask etherscan for help